prov/usnic/src/usdf_eq.c

```diff
@@ -194,11 +197,28 @@ static ssize_t usdf_eq_readerr(struct fid_eq *feq,
 		goto done;
 	}
 
-	ret = usdf_eq_read_event(eq, NULL, entry, sizeof(*entry), flags);
+	ret = usdf_eq_read_event(eq, NULL, &entry, sizeof(entry), flags);
+
+	/* read the user's setting for err_data. */
+	err_data = buffer->err_data;
+	err_data_size = buffer->err_data_size;
+
+	/* Copy the entry. */
+	*buffer = entry;
+
+	/* For compatibility purposes, if err_data_size is 0 on input, or the fab-
+	 * ric was opened with release < 1.5, err_data will be set to a data
```

Comment:
The comment mentions `< 1.5` but the conditional checks for `> 1.5`. You should probably change the comment to match the direction of the check.